subprojects/architecture-test/src/changes/archunit_store/internal-api-nullability.txt

```diff
@@ -6520,4 +6521,4 @@ Class <org.gradle.wrapper.PathAssembler$LocalDistribution> is not annotated (dir
 Class <org.gradle.wrapper.PathAssembler> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (PathAssembler.java:0)
 Class <org.gradle.wrapper.SystemPropertiesHandler> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (SystemPropertiesHandler.java:0)
 Class <org.gradle.wrapper.WrapperConfiguration> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (WrapperConfiguration.java:0)
-Class <org.gradle.wrapper.WrapperExecutor> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (WrapperExecutor.java:0)
\ No newline at end of file
+Class <org.gradle.wrapper.WrapperExecutor> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (WrapperExecutor.java:0)
```

Comment:
:x: What is the difference here?

Reply:
No idea what this was, went away after merging in `master`.